WBooks/Tabs/Library/LibraryController.swift

```diff
@@ -38,19 +38,19 @@ class MainMenuController: BaseViewController {
     
     private func configureNavigationBar() {
         navigationItem.title = "MAIN_MENU_NAV_BAR_TITLE".localized()
-        navigationItem.leftBarButtonItem = UIBarButtonItem(image: UIImage(named: "ic_notifications"), style: UIBarButtonItem.Style.plain, target: self, action: nil)
-        navigationItem.rightBarButtonItem = UIBarButtonItem(image: UIImage(named: "ic_search"), style: UIBarButtonItem.Style.plain, target: self, action: nil)
+        navigationItem.leftBarButtonItem = UIBarButtonItem(image: UIImage.notifications, style: UIBarButtonItem.Style.plain, target: self, action: nil)
+        navigationItem.rightBarButtonItem = UIBarButtonItem(image: UIImage.search, style: UIBarButtonItem.Style.plain, target: self, action: nil)
     }
 }
 
 // MARK: - UITableViewDelegate
-extension MainMenuController: UITableViewDelegate {
+extension LibraryController: UITableViewDelegate {
     func tableView(_ booksTable: UITableView, numberOfRowsInSection section: Int) -> Int {
```

Comment:
This methods are from the DataSource protocol =P
I guess you are not using the UITableViewDelegate protocol (for now...)